examples/ios/README.md

```diff
@@ -1,13 +1,16 @@
 # Flower iOS - A Flower SDK for iOS Devices with Example
-Flower iOS contains a Swift Package for Flower clients in iOS and an example that demonstrates a federated learning setup with iOS clients. The training is done on a MNIST dataset using CoreML with a updatable digit recognition model. 
+
+Flower iOS contains a Swift Package for Flower clients in iOS and an example that demonstrates a federated learning setup with iOS clients. The training is done on a MNIST dataset using CoreML with a updatable digit recognition model.
 
 ## Project Setup
 
 Requirements for setting up a server:
+
 - Conda for managing environment
```

Comment:
Is this a hard requirement? Because elsewhere we tend to use `pyenv` or `poetry`. `Poetry` might be the simplest one to use in this case. You might want to look at the setup we recommend in other examples, it's usually very basic. From the `quickstart_pytorch` example:
```bash
poetry install
poetry shell
```

Reply:
readme edited with poetry.